report/sections/Architecture/architecture.tex

```diff
@@ -12,19 +17,17 @@
 	\label{fig:Architecture}
 \end{figure}
 
-Note that the platform has an API running on a separate server instance to handle code execution and test, called \textit{Test Runner}. As components are introduced in the following sections, we elaborate on the choices made for each component, in relation to this structure.
-
 \section{Backend System}
-The backend system has several responsibilities including routing, data retrieval and updates and orchestration of execution of code tests. The backend is the link between all the components in the system. It operates as a traditional server in a client-server architecture.
+The backend system has several responsibilities including routing, data retrieval and updates and orchestration of execution of code tests. 
```

Comment:
```suggestion
The backend system has several responsibilities including routing, data retrieval and mutations, and orchestration of execution of code tests. 
```
We use mutations in other places + have described it earlier in the report as being similar to creating/updating/etc. data.